src/Core/Domain/Country/ValueObject/CountryId.php

```diff
@@ -64,7 +65,7 @@ public function getValue(): int
      */
     private function assertPositiveInt(int $value)
     {
-        if (0 > $value) {
+        if (NoStateId::NO_STATE_ID_VALUE >= $value) {
```

Comment:
`NoStateId` ? Shouldn't it be `NoCountryId` ?

Are you agree that you changed the behavior there ?
`0 > $value` => `0 >= $value`

Reply:
Indeed I didn't see this one but you should use `NoCountryId` here

Reply:
@MeKeyCool Yes i've change the behavior .. it's the aim of the PR ;-) But you're right for the contant error ! Well catched !